variants/pundit/template.rb

```diff
@@ -0,0 +1,46 @@
+source_paths.unshift(File.dirname(__FILE__))
+
+puts "Adding pundit to Gemfile"
+
+gem "pundit"
+
+run "bundle install"
+run "rails g pundit:install"
+run "rails g pundit:policy example"
+
+copy_file "spec/policies/example_policy_spec.rb", force: true
+
+# Configure app/controllers/application_controller.rb
+insert_into_file "app/controllers/application_controller.rb",
+                 after: /^class ApplicationController < ActionController::Base\n/ do
+  <<-RUBY
+    include Pundit
+
+    after_action :verify_authorized, except: %i[index], unless: :current_devise_controller?
+    after_action :verify_policy_scoped, only: :index, unless: :skip_policy_scoped_controller?
```

Comment:
Small thing but for consistency it might be nice to rename `:skip_policy_scoped_controller?` to `:skip_verify_policy_scoped?`

Reply:
I've changed it to `skip_verify_policy_scoped?`